src/EPR.Payment.Facade.UnitTests/Services/RegistrationFees/RegistrationFeesServiceTests.cs

```diff
@@ -114,5 +115,54 @@ public async Task CalculateProducerFeesAsync_HttpServiceThrowsException_ShouldLo
                     It.IsAny<Func<It.IsAnyType, Exception?, string>>()),
                 Times.Once);
         }
+
```

Comment:
I don't know when it got added but we have AutoMoqDataAttribute in both the EPR.Payment.Facade.UnitTests and EPR.Payment.Facade.Common.UnitTests. Originally, it was in EPR.Payment.Facade.Common.UnitTests so somebody copied the code to EPR.Payment.Facade.UnitTests. Remove it from EPR.Payment.Facade.UnitTests.

Also, do not use :
using AutoFixture.MSTest

Use it fro the namespace we have which is 
EPR.Payment.Facade.Common.UnitTests.TestHelpers

Change this file to use the correct namespace for AutoMoqdata attribute

Reply:
deleted from EPR.Payment.Facade.UnitTests